artemis-server/src/test/java/org/apache/activemq/artemis/tests/util/ActiveMQTestBase.java

```diff
@@ -276,6 +279,7 @@ public ActiveMQTestBase() {
 
    @After
    public void tearDown() throws Exception {
+      noProcessFilesBehind.tearDown();
```

Comment:
Why do you need a tearDown here? The Rule could clear itself.

Reply:
Some methods executed after tests ie shutdownDerby open new files (at the first execution) so I need a method called before to filter that files.